AutomatedTesting/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@
 #
 
 if(NOT PROJECT_NAME)
-    cmake_minimum_required(VERSION 3.20)
+    cmake_minimum_required(VERSION 3.24)
```

Comment:
I just noticed that Ubuntu 22.04 defaults to cmake 3.22, not 3.24. ( https://packages.ubuntu.com/jammy/cmake ). Is it possible to downgrade this to cmake 3.22 as the minimum instead?

Reply:
Yes, that should be ok. I'll change this and make a new announcement for that on Monday.